examples/06-verif-manual/vm-008-parametric_calculation.py

```diff
@@ -0,0 +1,216 @@
+
+kp = Create(100, 0, 30, -200, 25, 80)
+kp_dist, keypoint_list = kp.create_kp_method()
+print(f"Distance between keypoint is: {kp_dist:.2f}\n\n"
+      f"{keypoint_list}")
+
+
+###############################################################################
+# Distance between nodes.
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~
+# Using already created method for nodes to get the distance between them and
+# print out an output. The nodes have got next coordinates:
+#
+# * :math:`N_{\mathrm{1(x,y,z)}} = 1.5, 2.5, 3.5`
+# * :math:`N_{\mathrm{2(x,y,z)}} = -3.7, 4.6, -3`
+
+nodes = Create(1.5, 2.5, 3.5, -3.7, 4.6, -3)
+node_dist, node_list = nodes.create_node_method()
+print(f"Distance between nodes is: {node_dist:.2f}\n\n"
+      f"{node_list}")
+
+
+###############################################################################
+# Check Results
+# ~~~~~~~~~~~~~
+# Finally we have the results of the distances for both simulations,
+# which can be compared with expected target values:
+#
+# - 1st simulation to get the distance between keypoints :math:`K_3` and :math:`K_4`, where :math:`LEN_1 = 305.16\,(in)`
+# - 2nd simulation to get the distance between nodes :math:`N_1` and :math:`N_2`, where :math:`LEN_2 = 8.58\,(in)`
+#
+# For better representation of the results we can use ``pandas`` dataframe
+# with following settings below:
+
+# Define the names of the rows.
+row_names = ["N1 - N2 distance (LEN2)",
+             "K3 - K4 distance (LEN1)"]
+
+# Define the names of the columns.
+col_names = ['Target',
+             'Mechanical APDL',
+             'RATIO']
+
+# Define the values of the target results.
+target_res = np.asarray([8.5849, 305.16])
+
+# Create an array with outputs of the simulations.
+simulation_res = np.asarray([node_dist, kp_dist])
+
+# Identifying and filling corresponding columns.
+main_columns = {
+    "Target": target_res,
+    "Mechanical APDL": simulation_res,
+    "Ratio": list(np.divide(simulation_res, target_res))
+}
+
+# Create and fill the output dataframe with pandas.
+df2 = pd.DataFrame(main_columns, index=row_names)
+
+# Apply style settings for the dataframe.
+df2.style.set_table_styles([
+                            {
+                              "selector": "th",
+                              "props": [('font-size', '16px')]
+                            },
+                            {
+                              "selector": "td",
+                              "props": [('font-size', '16px')]
+                            },
+                            {
+                                "selector": "td:hover",
+                                "props": [("background-color", "#FFF8DC")]
+                            }]).set_properties(**
+                                               {
+                                                    "color": "black",
+                                                    "text-align": "center"
+                                               }).format("{:.2f}")
```

Comment:
```suggestion
df2.style
```

You could just use this, and the output is similar:
![image](https://user-images.githubusercontent.com/28149841/150503016-62c8566a-bf20-42cd-8432-154d243533fa.png)

But smaller font size.

Reply:
I really think this block of code is not needed.